docs/modules/ROOT/pages/explanation/pod-security-admissions.adoc

```diff
@@ -0,0 +1,103 @@
+= Pod Security Admissions
+
+Kubernetes 1.24 removes the Pod Security Policy feature in favor of Pod Security Admission. This document explains the impact on {product} users.
+
```

Comment:
Preferably, add a tl;dr

1. Check PSA warnings
2. Set security context accordingly in your workload resources.